[anomaly detection] explain how anomaly monitors are configured via the API (#7900)
DataDog/documentation: content/en/monitors/monitor_types/anomaly.md

## API

Enterprise-level customers can create anomaly detection monitors using the [create-monitor API endpoint][9]. Datadog **strongly recommends** [exporting a monitor's JSON][10] to build the query for the API. By using the [monitor creation page][1] in the Datadog, customers benefit from the preview graph and automatic parameter tuning to help avoid a poorly configured monitor.

**Note**: Anomaly detection monitors are only available to enterprise-level customers. Pro-level customers interested in anomaly detection monitors should reach out to their customer success representative or email the [Datadog billing team][11].

> Anomaly monitors are managed via the [same API][12] as other monitors. However, there are some fields in the request that are unique for anomaly monitors and are discussed in more detail below.

Reviewer: ```suggestion
Anomaly monitors are managed using the [same API][12] as other monitors. These fields are unique for anomaly monitors:
```

* `direction`: the directionality of anomalies that should trigger an alert; `above`, `below`, or `both`
* `alert_window`: the timeframe which will be checked for anomalies (e.g., `last_5m`, `last_1h`)
* `interval`: a positive integer representing the number of seconds in the rollup interval; we recommend that the `interval` be at least a fifth of the `alert_window` duration
* `default_zero`: use `true` for most monitors; only set to `false` if submitting a count metric in which the lack of a value should _not_ be interpreted as a zero
* `seasonality`: `hourly`, `daily`, or `weekly`; exclude this parameter when using the `basic` algorithm
* `threshold`: a positive number no larger than 1; the fraction of points in the `alert_window` that must be anomalous in order for a critical alert to trigger

Below is an example query for an anomaly detection monitor, which alerts when the average Cassandra node's CPU is three standard deviations above the ordinary value over the last 5 minutes:

```text
avg(last_1h):anomalies(avg:system.cpu.system{name:cassandra}, 'basic', 3, direction='above', alert_window='last_5m', interval=20, count_default_zero='true') >= 1
```

### `options`

> Most of the properties in `options` property in the request body are the same as for other query alerts, but special attention should be given to `thresholds` and `threshold_windows`.

Reviewer: ```suggestion
Most of the properties under `options` in the request body are the same as for other query alerts, except for `thresholds` and `threshold_windows`.
```

> #### `thresholds`

Author: hmm. Due to the header 4 styling, this shows up as `THRESHOLDS`. Would it be better to not make this a heading or make it header 5?

Reviewer: yeah, good catch, made a suggestion to make it bold instead of h4

> #### `thresholds`

Reviewer: ```suggestion
**`thresholds`**
```

Anomaly monitors support `critical`, `critical_recovery`, `warning`, and `warning_recovery` thresholds. Thresholds are expressed as numbers from 0 to 1, and are interpreted as the fraction of the associated window that is anomalous. For example, an `critical` threshold value of `0.9` means that a critical alert triggers when at least 90% of the points in the `trigger_window` (described below) are anomalous. Or, a `warning_recovery` value of 0 means that the monitor recovers from the warning state only when 0% of the points in the `recovery_window` are anomalous.

The `critical` `threshold` should match the `threshold` used in the `query`.

> #### `threshold_windows`

Reviewer: ```suggestion
**`threshold_windows`**
```
